packages/filecoin-api/src/aggregator/events.js

```diff
@@ -324,6 +325,42 @@ export const handleInclusionInsertToIssuePieceAccept = async (
   return { ok: {} }
 }
 
+/**
+ * Handle issueing inclusion claims once piece is included in an aggregate.
+ *
+ * @param {import('./api.js').InclusionInsertEventToIssueInclusionClaim} context
+ * @param {import('./api.js').InclusionRecordWithProof} record
+ */
+export const handleInclusionInsertToIssueInclusionClaim = async (
+  context,
+  record
+) => {
+  const claimResult = await Assert.inclusion
+    .invoke({
+      issuer: context.assertService.invocationConfig.issuer,
+      audience: context.assertService.invocationConfig.audience,
+      with: context.assertService.invocationConfig.with,
+      nb: {
+        content: record.aggregate,
+        includes: record.piece,
+        proof: record.proof,
```

Comment:
What is proof here?

Reply:
proof is the CID of the dag-cbor block that we encode the proof on for storing it

Reply:
https://us-west-2.console.aws.amazon.com/dynamodbv2/home?region=us-west-2#item-explorer?operation=SCAN&table=prod-w3filecoin-aggregator-inclusion-store the inclusion store is storing it map, to the proof bytes